src/Symfony/Component/Console/Application.php

```diff
@@ -901,6 +901,32 @@ protected function doRenderThrowable(\Throwable $e, OutputInterface $output): vo
         } while ($e = $e->getPrevious());
     }
 
+    /**
+     * Get shell verbosity.
+     *
+     * @param InputInterface $input
+     * @return int
+     */
+    public function getShellVerbosity(InputInterface $input): int
```

Comment:
Can you cover this with some tests please?!

Reply:
Added tests at `ApplicationTest::testGetShellVerbosity`